src/bikeracks/__init__.py

```diff
@@ -53,10 +53,12 @@ def create_app(test_config=None):
         pass
     
     # Create error logging file
+    log_dir_path = os.path.join(app.instance_path, 'logs')
+    error_log_file = os.path.join(app.instance_path, 'logs/errors.log')
```

Comment:
`join(app.instance_path, 'logs', 'errors.log')`

Reply:
Okay, I changed it.